includes/gateway/class-omise-payment-mobilebanking.php

```diff
@@ -85,7 +85,7 @@ public function charge($order_id, $order)
 		);
 
 		$requestData['source'] = array_merge($requestData['source'], [
-			$requestData['platform_type'] = Omise_Util::get_platform_type(wc_get_user_agent())
```

Comment:
So in the previous version, was it generating a source object like the following?

```
{
  "type": "mobile_banking",
  "0": "IOS"
}
```

Reply:
Like the following to be exact.

```
"source_type" : {
  "type": "mobile_banking",
  "0": "IOS"
},
"platform_type": "IOS"
```